### DISCUSSION TITLE
ERR_SSL_UNSAFE_LEGACY_RENEGOTIATION_DISABLED

### DISCUSSION THREAD

**Discussion by AltugEngin** [pockethost/pockethost#252] [Q&A] [answered]
Hello
My sveltekit project with the following configuration gives me following error. 
Is this an https issue rather http? I'm complete lost. Pocketbase run perfect locally but i could not success deploying it

```
const url = 'https://undefined.pockethost.io';
export const pb = new PocketBase(url)
```

```
library: 'SSL routines',
reason: 'unsafe legacy renegotiation disabled',
code: 'ERR_SSL_UNSAFE_LEGACY_RENEGOTIATION_DISABLED'
```

**Comment by benallfree** [comment]
@AltugEngin This is a new one to me. Why does it say `undefined` in the URL?

**Reply by AltugEngin** [reply]
Sorry for the confusion. The exact link is
"https://gebze-elektrik.pockethost.io" but same error occurs anyway.

**Reply by benallfree** [reply]
Are you seeing the error in the Chrome debugger?

**Reply by AltugEngin** [reply]
No. Error displayed on vscode node terminal. i run app in my local development environment. İs this just because i try this company network? Maybe some restrictions occured. Have a try another network

**Reply by benallfree** [reply]
From my research, it appears that node 18+ disables SSL renegotiation because it's unsafe. Did you follow the instructions here https://github.com/pocketbase/js-sdk#nodejs-via-npm?

**Reply by AltugEngin** [reply]
What i understand is the options available are:


(1) downgrade my node <18,

(2) https://stackoverflow.com/questions/72866798/node-openssl-legacy-provider-is-not-allowed-in-node-options/76161893#76161893

(i didnt try this option yet)

**Reply by AltugEngin** [reply]
Adding below option in my .npmrc file doesn't help
`node-options="--openssl-legacy-provider"`

**Reply by benallfree** [reply]
Can you create a quick repo or codesandbox project that reproduces the problem you're having? I'd like to understand more about what's happening.

**Reply by AltugEngin** [reply, marked as answer]
Thank you Ben! I'm now able to connect to the database from another network with **Node v18.16.** (That error was coming from our stupid company network. Seems network security issue) 
This solves my issue at the moment. Thank you again.
